### PR TITLE
Receive view improvements

### PR DESCRIPTION
**Pull request by guilhermemntt**
Closes https://github.com/decred/decrediton/issues/2450.

This PR ignores the share button, since Electron does not support this native functionality (see https://github.com/decred/decrediton/issues/2450#issuecomment-626497410). By the way, the amount input has been kept, so feel free to close it, due the fact the implemented PR is very different from the provided specs.

### CODE REVIEW

**Comment by guilhermemntt** [comment]
Thanks for the feedback! The Decred icon has been re-added.

**Comment by guilhermemntt** [comment]
Rebased.

**Comment by alexlyp** [comment]
Needs a rebase.

**Comment by guilhermemntt** [comment]
Rebased.

**Comment by alexlyp** [comment]
Icons needs a rebase once again.

File: app/components/views/TransactionsPage/ReceiveTab/ReceiveTabPage.jsx [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+        </div>
+
+        <div className={style.line}>
+          <div
+            className={
+              style.receiveContentNestQR + " " + (tooltip && style.border)
+            }>
+            {nextAddress}
+            <div
+              className={
```
Comment (amassarwi) [inline app/components/views/TransactionsPage/ReceiveTab/ReceiveTabPage.jsx:81]: use classnames function

File: app/components/views/TransactionsPage/ReceiveTab/ReceiveTabPage.jsx [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+              onChangeAmount={(e) => onValidateAmount(e)}
+              onKeyDown={onKeyDown}
+            />
+          </div>
+        </div>
+
+        <div className={style.line}>
+          <div
+            className={
+              style.receiveContentNestQR + " " + (tooltip && style.border)
```
Comment (amassarwi) [inline app/components/views/TransactionsPage/ReceiveTab/ReceiveTabPage.jsx:77]: classnames function!

File: app/components/views/TransactionsPage/ReceiveTab/ReceiveTabPage.jsx (hunk 1 above)
Comment (guilhermemntt) [reply app/components/views/TransactionsPage/ReceiveTab/ReceiveTabPage.jsx]: My bad. Thanks for the review!

File: app/components/views/TransactionsPage/ReceiveTab/ReceiveTabPage.jsx (hunk 2 above)
Comment (guilhermemntt) [reply app/components/views/TransactionsPage/ReceiveTab/ReceiveTabPage.jsx]: Fixed too.

**Comment by amassarwi** [comment]
@guilhermemntt Thanks 👍

**Comment by guilhermemntt** [comment]
Rebased.

**Comment by alexlyp** [comment]
Needs a rebase plz

**Comment by guilhermemntt** [comment]
Rebased.

**Pull request merged**